hera_qm/xrfi.py

```diff
@@ -1398,6 +1398,7 @@ def xrfi_run_step(uv_files=None, uv=None, uvf_apriori=None,
                    dtype='uvcal', run_filter=True,
                    metrics=None, flags=None, modified_z_score=False,
                    a_priori_flag_yaml=None,
+                   a_priori_ants_only=False,
```

Comment:
I'd suggest making this default True, since we know that it can create problems when its False.

Reply:
I'm leaving the default to False because this behavior is less confusing and the problems being created are not inherent to apriori flags but rather their interaction with the median filtering strategy (in a way that should be similar to edge effects?). 

I have added a warning though that spurious over-flagging can occur within a kernel width of the flags.